extensions/bitwarden/src/api.ts

```diff
@@ -0,0 +1,38 @@
+import execa from "execa";
+
+export class BitwardenApi {
+  private env: Record<string, string>;
+  constructor(clientId: string, clientSecret: string) {
+    this.env = {
+      BW_CLIENTSECRET: clientSecret,
+      BW_CLIENTID: clientId,
+    };
+  }
+
+  async syncItems(sessionToken: string): Promise<void> {
+      await this.exec(["sync", "--session", sessionToken])
+  }
+
+  async login(): Promise<void> {
+     await this.exec(["login", "--apikey"]);
+  }
+
+  async listItems<ItemType>(type: string, sessionToken: string): Promise<ItemType[]> {
+    const {stdout} = await this.exec(["list", type, "--session", sessionToken])
+    return JSON.parse(stdout)
+  }
+
+  async unlockVault(password: string): Promise<string> {
```

Comment:
```suggestion
  async unlock(password: string): Promise<string> {
```
This isn't limited to vaults - it applies to account sessions.